ecphp/api-gw-authentication-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "EcPhp\\ApiGwAuthenticationBundle\\ApiGwAuthenticationBundle": ["all"]
+    },
+    "copy-from-package": {
+        "Resources/config/": "%CONFIG_DIR%/"
+    }
```

Comment:
I dont think adding recipes like this is a good idea. 

See the best practices in the readme: https://github.com/symfony/recipes-contrib#best-practices

Reply:
I understand you just wrote those new best-practices after seeing my PR (_thanks for sending me the link of [your PR](https://github.com/symfony/recipes-contrib/pull/1093) as kindly requested on Slack by the way_).

All the previous PR for my other ECPHP packages were accepted straight, by you: https://github.com/symfony/recipes-contrib/pulls?q=is%3Apr+is%3Aclosed+ecphp

Today [a new Best Practice paragraph](https://github.com/symfony/recipes-contrib/pull/1093), I read:

> The symfony/recipes-contrib repository should contain the config for packages. Using "copy-from-package Configurator" for routes and config is not allowed. That would make the recipes impossible to maintain and to assure their quality.

Now, if I provide the same PR with the config files that I automatically copy, what would it change?
Do you really think that just by reading those files you can correctly or better assert/maintain/evaluate the quality of a bundle?
The idea of having such a recipe is to facilitate the installation of configuration files, nothing else, how could you assert the quality of a bundle just by reading those files?

Thanks.

Reply:
> I understand you just wrote those new best-practices after seeing my PR

That is correct that I just wrote them. But they have been discussed since last year. 

Im sorry that I missed sending you the link. 

> All the previous PR for my other ECPHP packages were accepted straight, by you: https://github.com/symfony/recipes-contrib/pulls?q=is%3Apr+is%3Aclosed+ecphp

That is correct. Recipes like ECPHP was a part of the reason why I thought we needed some better guidance. 

> Now, if I provide the same PR with the config files that I automatically copy, what would it change?

Yes

> Do you really think that just by reading those files you can correctly or better assert/maintain/evaluate the quality of a bundle?

Yes

> The idea of having such a recipe is to facilitate the installation of configuration files, nothing else, how could you assert the quality of a bundle just by reading those files?

I can be sure that this recipe does not violate any of the other quality rules.

Reply:
After discussing this matter with @nicolas-grekas, I decided to submit the request changes, even if I do not agree with this new best-practice rule.

Reply:
Thank you